--- sshd_config	2022-11-02 04:29:32.000000000 +0800
+++ sshd_config.patched	2022-12-02 15:22:01.609319775 +0800
@@ -28,11 +28,11 @@
 
 # Authentication:
 
-#LoginGraceTime 2m
+LoginGraceTime 10s
 #PermitRootLogin prohibit-password
 #StrictModes yes
-#MaxAuthTries 6
-#MaxSessions 10
+MaxAuthTries 2
+MaxSessions 4
 
 #PubkeyAuthentication yes
 
@@ -54,7 +54,7 @@
 #IgnoreRhosts yes
 
 # To disable tunneled clear text passwords, change to no here!
-#PasswordAuthentication yes
+PasswordAuthentication no
 #PermitEmptyPasswords no
 
 # Change to no to disable s/key passwords
@@ -103,7 +103,7 @@
 #VersionAddendum none
 
 # no default banner path
-#Banner none
+Banner /etc/banner
 
 # override default of no subsystems
 Subsystem	sftp	/usr/lib/ssh/sftp-server
